app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Material.kt

```diff
@@ -6,15 +6,15 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 /**
- * @param mid: DB ID for dev use
+ * @param mId: DB ID for dev use
  * @param materialName: User Inputted Name
  * @param materialTypeId: Type of material from predetermined list
  * @param brandId: Brand from brand object
  * @param materialColor: Color of filament. Uses android color ints
  * @param materialAmount: Amount of filament in g or L
  */
 @Entity(tableName = "material")
-data class Material (@PrimaryKey var mid: Int,
+data class Material (@PrimaryKey var mId: Int,
```

Comment:
Ditto comment above.  As long as you're in here, name it materialId.  materialID is also acceptable, as there's not a well defined convention for acronyms.